org.qownnotes.QOwnNotes.json

```diff
@@ -26,9 +26,10 @@
                     "url": "https://download.tuxfamily.org/qownnotes/src/qownnotes-23.6.6.tar.xz",
                     "sha256": "517127a1ef13a2fce084ab445c38ace4693dde80c4c8986135732719a2b95572",
                     "x-checker-data": {
-                        "type": "html",
-                        "url": "https://download.tuxfamily.org/qownnotes/src/",
-                        "pattern": "(qownnotes-([\\d\\.]+).tar.xz)",
+                        "type": "json",
+                        "url": "https://api.github.com/repos/pbek/QOwnNotes/releases/latest",
+                        "version-query": ".tag_name",
+                        "url-query": ".assets | first | .browser_download_url",
```

Comment:
I wonder when that will break, when the source-archive isn't the first URL anymore in the API result...

Reply:
Wouldn't it be more safe to also use a pattern like `"pattern": "(qownnotes-([\\d\\.]+).tar.xz)",` (if that's possible)?

Reply:
"pattern": is invalid attribute here, we need to use something else.